mungegithub/mungers/assign-unassign-handler_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"runtime"
+	"testing"
+
+	"k8s.io/contrib/mungegithub/github"
+	github_test "k8s.io/contrib/mungegithub/github/testing"
+	"k8s.io/kubernetes/pkg/util/sets"
+
+	goGithub "github.com/google/go-github/github"
+)
+
+var (
+	testUsername = "test-user"
+	testUser     = goGithub.User{ID: intPtr(-1), Name: &testUsername}
+)
+
+func TestAssignComment(t *testing.T) {
+	runtime.GOMAXPROCS(runtime.NumCPU())
+
+	tests := []struct {
+		testName          string
+		comments          []*goGithub.IssueComment
+		existingAssignees []*goGithub.User
+		newAssignees      sets.String
+		removedAssignees  sets.String
+	}{
+		{
+			testName: "Assign cmd should add to existing assignees",
```

Comment:
nit: Test name should probably be changed to avoid confusion with the next test, which has the same name.